toolsWebsite/tools/templates/tools/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+
+{% load static %}
+<html lang="zxx">
+
+<head>
+  <meta charset="utf-8">
+
+  <title>Index</title>
+  <link rel="stylesheet" href="{% static 'tools/styles.css' %}" type="text/css"/>
+
+</head>
+
+<body>
+  <h1>List of Tools</h1>
+{% for category in tools_list %}
+  <br>
+  <h4 style="font-size: large">{{ category.type }}</h4>
+  <p style="font-size: large">Available: {{ category.count }}</p>
```

Comment:
Wouldn't the number of available tools be `category.count - category.unavailable`?

Reply:
Count was a bad variable name.  Count is actually the number of tools available.  (i.e. there could be 3 available and 5 checked out)